src/index.js

```diff
@@ -0,0 +1,10 @@
+const app = require('./app');
```

Comment:
I did it because jest gave an error like Jest did not exit one second after the test run has completed. So after removing the connection to another file fixed it. But now I found a better way. I exported the server connection and close it on the test after the test are completed.